src/patterns/PhoneNumberInput/PhoneNumberInput.tsx

```diff
@@ -0,0 +1,115 @@
+ *
+ * [Storybook Link](https://phillips-seldon.netlify.app/?path=/docs/patterns-PhoneNumberInput--overview)
+ */
+const PhoneNumberInput = forwardRef<HTMLDivElement, PhoneNumberInputProps>(
+  (
+    {
+      className,
+      value = '',
+      countryCode = '',
+      onChange,
+      label = 'Phone',
+      required = false,
+      error,
+      errorText,
+      disabled = false,
+      ...props
+    },
+    ref,
+  ) => {
+    const { className: baseClassName, ...commonProps } = getCommonProps(props, 'PhoneNumberInput');
+    const [selected, setSelected] = React.useState(countryCode || undefined);
+
+    // Find the country object by code for display
+    const selectedCountry = countries.find((c) => c.code === selected);
+
+    // Handle phone number input change
+    const handleInputChange = (e: React.ChangeEvent<HTMLInputElement>) => {
+      onChange?.(e.target.value, selected || '');
+    };
+
+    const errorId = errorText ? `${baseClassName}-error-text` : undefined;
+
+    return (
+      <div ref={ref} className={classnames(`${baseClassName}`, className)} {...commonProps}>
+        <div className={`${baseClassName}__wrapper`}>
+          <div className={`${baseClassName}__country-picker`}>
+            <input type="hidden" name="phoneNumber" value={value} data-testid="phone-number-hidden-input" />
+            <input
+              type="hidden"
+              name="phoneCountryCode"
+              value={selected || ''}
+              data-testid="phone-country-code-hidden-input"
+            />
+            <CountryPicker
+              triggerLabelText="Phone Number"
+              triggerDisplayValue={selectedCountry ? `+${getSafeCountryCallingCode(selectedCountry.code)}` : ''}
+              hasTriggerError={!!error}
+              modalTitle="Country code"
+              searchInputLabel=""
+              searchInputPlaceholder="Search country"
+              selectButtonLabel="Select"
+              isPhone={true}
+              value={selectedCountry?.code}
+              onChange={setSelected}
+            />
+          </div>
+          <div className={`${baseClassName}__input`}>
+            <Input
+              id="phone-input"
+              type="tel"
+              labelText={label}
+              hideLabel
+              value={value}
+              onChange={handleInputChange}
+              required={required}
+              invalid={!!error}
+              disabled={disabled}
+              // We don't want to use the input default text, and the error text will be shown under the country picker
+              invalidText=""
+              aria-describedby={errorId}
+            />
+          </div>
+        </div>
+        {/** Mimics the Input component validation */}
+        {errorText ? (
+          <Text className={classnames(`${baseClassName}__error`, `${baseClassName}__error-msg`)} id={errorId}>
```

Comment:
Small nitpick but since these classes are not conditional we can just add them without the classname method.